lighthouse-core/gather/driver.js

```diff
@@ -287,7 +288,7 @@ class Driver {
             }
 
             // If both the data and the url are empty strings, the page had no manifest.
-            return reject('No web app manifest found.');
+            return reject(ExpectedError(new Error('No web app manifest found.')));
```

Comment:
(and this whole method will hopefully get fixed in https://github.com/GoogleChrome/lighthouse/pull/2349#discussion_r118798623)